misc/grimoirelab2sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Luis Cañas-Díaz <lcanas@bitergia.com>
+#
+
+from __future__ import unicode_literals
+
+import argparse
+import datetime
+import json
+import sys
+
+from sortinghat.exceptions import InvalidFormatError
+from sortinghat.parsing.grimoirelab import GrimoireLabParser
+
+
+GRIMOIRELAB2SH_DESC_MSG = \
+"""Export identities information from GrimoireLab files to Sorting Hat
+JSON format."""
```

Comment:
Put the whole docstring in the same line, otherwise it will show a weird message when help command is called. By the way, we don't have problems with lines longer than 80 characters, so feel the freedom :-)